Makefile

```diff
@@ -44,6 +44,10 @@ fmt:
 vet:
 	go vet ./pkg/... ./cmd/...
 
+# Run go fmt Test
+check-gofmt:
+	test -z $(shell gofmt -l . | grep -v ^vendor)
```

Comment:
please make sure there are double quotes `"` around the `$(...)`.

Reply:
Done